### ISSUE TITLE
Investigate how to make a timeout/popup copy accessible

### ISSUE THREAD

**Issue by dnoneill** [sul-dlss/sul-embed#2786]
EarthWorks [Code Snippets](https://earthworks.stanford.edu/catalog/stanford-pb606mt4669) switches from `Copy` to `Copied` when clicked, but this isn't announced to screen readers. [Aria-live](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-live) might be relevant.

**Comment by alundgard** [comment]
@corylown It seems to announce on Safari but not on Chrome?

**Comment by corylown** [comment]
Thanks @alundgard I'll test on Chrome.

**Comment by corylown** [comment]
Each browser has different behavior with voiceover when the label or text changes for a button with focus.

Firefox: announces the changed text if the button is selected with `enter` or `ctrl-option-space`
Chrome: only announces the changed text if the button is selected with `enter`
Safari: only announce the changed text if the button is selected with `ctrl-option-space`

I do not think it is appropriate to use `aria-live` on the button itself because its purpose is to draw attention to changes on the page that do not have current focus. Furthermore, adding `aria-live` to the button will announce the change _twice_ depending on the browser and how the button was selected (`enter` `ctrl-option-space`).

It _might_ be appropriate to add a visually-hidden alert after the button to which we add a message "Copied to clipboard" that would be announced to the screen reader:

```
<span aria-live="polite" class="visually-hidden" role="alert" data-clipboard-copy-target="successMessage"></span>
```
the clipboard_copy_controller.js could update this with the message we want announced:
```
this.successMessageTarget.innerHTML = "Copied to clipboard";
```

**Comment by alundgard** [comment]
For posterity, further discussion and findings in this thread: https://stanfordlib.slack.com/archives/C07A2R7EE/p1742300408933109

**Comment by dnoneill** [comment]
@corylown Is this okay to close?
